data/test-cases/plugins/ms-teams-sync/ui-integration/List_linked_channels.md

```diff
@@ -0,0 +1,51 @@
+---
+# (Required) Ensure all values are filled up
+name: "List of linked channels in RHS"
+status: Active
+priority: Normal
+folder: ui-integration
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Click on MS Teams sync icon in App bar or channel header and open the RHS when there are MM channels linked to MS Teams channel.
+
+**Step 2**
+
+1. Click on MS Teams sync icon in App bar or channel header and open the RHS when there are MM channels linked to MS Teams channel and the logged-in user is a `normal user`.
```

Comment:
I don't think normal user is the correct term here. We should update this to something else.

Reply:
then i think we should update it as member

Reply:
i think we can use a term something like member having admin permissions and member having non-admin permissions